.github/workflows/verify_base_branch.yaml

```
@@ -0,0 +1,40 @@
+name: Pull Request base branch verification
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
```

Make git happy when type list changes.
```suggestion
    types:
        - opened
        - synchronize
        - reopened
        - edited
```